org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/internal/gradle/checksums/WrapperValidator.java

```diff
@@ -114,6 +130,9 @@ public String apply(Map<String, String> input) {
 					File cacheDir = getSha256CacheFile();
 					for (String wrapperChecksumUrl : wrapperChecksumUrls) {
 						try {
+							if (wrapperChecksumUrls.contains(wrapperChecksumUrl)) {
```

Comment:
always true since that's what you're iterating over

Reply:
Fixed.